pkg/handlers/system_intake_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/google/uuid"
 	"github.com/gorilla/mux"
 	"github.com/guregu/null"
+	"github.com/pkg/errors"
```

Comment:
What's this for? Doesn't the stdlib `errors` do everything you need?

Reply:
wow, great catch. I never would have spotted this! I wonder how this happened. Goland is probably the culprit